lib/web/apiserver.go

```diff
@@ -4284,6 +4292,10 @@ func (h *Handler) fetchExistingSession(ctx context.Context, clt authclient.Clien
 		return session.Session{}, nil, trace.Wrap(err)
 	}
 
+	if types.IsOpenSSHNodeSubKind(tracker.GetTargetSubKind()) {
+		return session.Session{}, nil, trace.BadParameter("session joining is only supported for Teleport nodes, not OpenSSH nodes")
```

Comment:
nit: openssh nodes can be considered Teleport nodes, this can be ambiguous

```suggestion
		return session.Session{}, nil, trace.BadParameter("session joining is only supported for nodes which are Teleport agents, not OpenSSH nodes")
```